CMakeInstallation.cmake

```diff
@@ -364,7 +364,7 @@ if (NOT HDF4_EXTERNALLY_CONFIGURED AND NOT HDF4_NO_PACKAGES)
     set (CPACK_PACKAGING_INSTALL_PREFIX "/${CPACK_PACKAGE_INSTALL_DIRECTORY}")
     set (CPACK_PACKAGE_ICON "${HDF_RESOURCES_EXT_DIR}/hdf.icns")
 
-    option (HDF4_PACK_MACOSX_FRAMEWORK  "Package the HDF Library in a Framework" OFF)
+    option (HDF4_PACK_MACOSX_FRAMEWORK  "Package the HDF Library in a Frameworks" OFF)
```

Comment:
Is "Frameworks" correct here? 'a' is singular.

Reply:
IDK, but it is used everywhere.